timur/lib/server/controllers/timur_controller.rb

```diff
@@ -17,7 +17,8 @@ def config_json
       {
         project_name: @params[:project_name],
         token_name: Timur.instance.config(:token_name),
-        magma_host: Timur.instance.config(:magma)[:host],
+        magma_host: Timur.instance.config(:magma)&.dig(:host),
+        metis_host: Timur.instance.config(:metis)&.dig(:host),
         metis_uid_name: Timur.instance.config(:metis_uid_name),
         janus_host: Timur.instance.config(:auth_redirect)
```

Comment:
Since you're adding a `:janus:` `:host:` explicitly to the `config.yml` file, does it make sense to use that here instead of `auth_redirect`?